### PR TITLE
Add hope to showcase

### PR DESCRIPTION
**Pull request by imcmai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ A curated list of awesome things related to [docsify](https://docsify.js.org)
 - [JS-Ninja](https://abpanic.github.io/Mastering-JavaScript/#/) - JavaScript course to start as begineer and upskill to clear staff/principal level interviews.
 - [Sakura Docs](https://sakura-docs.arkinsolomon.net) - A programming language designed for package installers.
 - [Ransomware.live](https://www.ransomware.live) - A monitoring ransomware's victims in near real-time. 
-
+- [hope](https://cmai.xin/#/) - A blog documenting.
```
Comment (sy-records) [inline README.md:144]:
```suggestion
- [hope](https://cmai.xin/#/) - A blog documenting.

```

**Review by sy-records — CHANGES_REQUESTED** [review]

**Review by sy-records — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -141,6 +141,7 @@ A curated list of awesome things related to [docsify](https://docsify.js.org)
 - [JS-Ninja](https://abpanic.github.io/Mastering-JavaScript/#/) - JavaScript course to start as begineer and upskill to clear staff/principal level interviews.
 - [Sakura Docs](https://sakura-docs.arkinsolomon.net) - A programming language designed for package installers.
 - [Ransomware.live](https://www.ransomware.live) - A monitoring ransomware's victims in near real-time. 
+- [hope](https://cmai.xin/#/) - A blog documenting.
```
Comment (trusktr) [inline README.md:144]: A blog documenting what? This is an incomplete sentence. May as well have no description.

Comment (sy-records) [reply README.md:144]: I checked the content and maybe `A blog documenting my growth process.` would be more appropriate.

Comment (imcmai) [reply README.md:144]: Thanks

**Pull request merged**